project/plugins.sbt

```diff
@@ -1,3 +1,18 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
+//Create a fat JAR of your project with all of its dependencies.
```

Comment:
I think scala check is still missing, you want to check with @jyoti once as we have decided to add conf files and the plugins from Spotify.
pls also refer this - https://spotify.github.io/scio/dev/Style-Guide.html